brave/brave-core
# Updated zoom constants and pages to resolve issue #6943

Fix https://github.com/brave/brave-browser/issues/6943

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist. - see https://github.com/brave/brave-browser/issues/6943
- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Verify you can set Zoom level in brave://settings to 133% and 140%
2. Open a PDF
3. Verify you can use Zoom level 133% and 140% (icon shows in omnibar)
4. Try to print the page
5. Print preview will come up
6. Verify you can zoom to 133% and 140%

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
